backend/src/main/java/me/quizzl/backend/controllers/QuizController.java

```diff
@@ -28,4 +29,9 @@ public void addQuiz(@RequestBody Quiz quiz){
     public List<Quiz> getQuizzes(){
         return quizService.getQuizzes();
     }
+
+    @GetMapping("/{id}")
+    public Quiz getQuiz(@PathVariable String id){
+        return quizService.getQuizByID(UUID.fromString(id));
+    }
```

Comment:
if you do modify `getQuizByID`, it'll likely end up returning an `Optional<Quiz>` instead. in that case, you could do something like this to get this endpoint to return a 404 when a quiz by that ID isn't found:
```suggestion
    @GetMapping("/{id}")
    public ResponseEntity<Quiz> getQuiz(@PathVariable String id){
        var quiz = quizService.getQuizByID(UUID.fromString(id));
        if (quiz.isPresent()) {
            return new ResponseEntity<Quiz>(quiz.get(), HttpStatus.OK);
        }
        return new ResponseEntity<Quiz>(HttpStatus.NOT_FOUND);
    }
```